DOCS/request-new-license.md

```diff
@@ -3,28 +3,29 @@ The SPDX Legal Team appreciates proposals for new free and open source licenses
 
 # How to request a new license or exception be added to the SPDX License List
 
-1. Join the [legal team mailing list](https://lists.spdx.org/g/spdx-legal).
+1.  Review the [license inclusion principles](license-inclusion-principles.md). Please refrain from submitting licenses that clearly do not meet these principles, for example, a license for which the text is not finalized or is only used on your personal project.
```

Comment:
For the end of this, how would you feel about something like "...or is a non-FOSS license only used for a small or personal project"?

A "personal" project could potentially have a license that is appropriate to add, if e.g. the license meets FOSS definitions and that personal project has some significant use, or something like that.

Reply:
good call